run_unittests.py

```diff
@@ -3113,6 +3113,12 @@ def _test_stds_impl(self, testdir, compiler, p):
                 (version_compare(compiler.version, '<5.0.0') or
                  (compiler.clang_type == mesonbuild.compilers.CLANG_OSX and version_compare(compiler.version, '<9.2')))):
                 continue
+            if (compiler.get_id() == 'clang' and '2a' in v and
+                (version_compare(compiler.version, '<6.0.0') or
+                 (compiler.clang_type == mesonbuild.compilers.CLANG_OSX and version_compare(compiler.version, '<9.2')))):
```

Comment:
I copied the appleclang version for the c++17 feature here, but 9.1 accepts both of these flags (`-std=c++17` and `-std=c++2a`). Should these both be 9.1 instead of 9.2?